tests/remotes/gdrive.py

```diff
@@ -38,7 +38,8 @@ def get_url():
 
 
 @pytest.fixture
-def gdrive(make_tmp_dir):
+def gdrive(test_config, make_tmp_dir):
+    test_config.requires("gdrive")
```

Comment:
To enable this feature on a fixture level, you need to use `test_config` and call `test_config.requires(remote) to skip them or run them.